ahmed-yara-sara/src/components/MovieItem.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from "react";
+import { Card, Button, Col } from "react-bootstrap";
+import MovieGrid from "./MovieGrid";
+function MovieItem(props) {
+  return (
+    <>
+      <Col>
+        <Card className="m-5 p-2" style={{ width: "300px" }}>
+          <Card.Img
+            variant="top"
+            src={`https://image.tmdb.org/t/p/w500/${props.image}`}
+          />
+          <Card.Body>
+            <li>{props.title}</li>
+            <li>{props.overview}</li>
+            <li>{props.popularity}</li>
+            <li>{props.release_date}</li>
+          </Card.Body>
+        </Card>
+      </Col>
+    </>
+  );
+}
```

Comment:
if you do the destructure thing that will make a different in your code readability I think.

```suggestion
function MovieItem(props) {
const { title, overview, popularity, release_date} = props
  return (
    <>
      <Col>
        <Card className="m-5 p-2" style={{ width: "300px" }}>
          <Card.Img
            variant="top"
            src={`https://image.tmdb.org/t/p/w500/${props.image}`}
          />
          <Card.Body>
            <li>{title}</li>
            <li>{overview}</li>
            <li>{popularity}</li>
            <li>{release_date}</li>
          </Card.Body>
        </Card>
      </Col>
    </>
  );
}
```